solver/src/settlement_submission/archer_settlement.rs

```diff
@@ -385,7 +385,10 @@ mod tests {
     #[tokio::test]
     #[ignore]
     async fn mainnet_settlement() {
-        shared::tracing::initialize("solver=debug,shared=debug,shared::transport::http=info");
+        shared::tracing::initialize(
+            "solver=debug,shared=debug,shared::transport::http=info",
+            "off".parse().unwrap(),
```

Comment:
If we use `Option<Level>`, then this can just be:
```suggestion
            None,
```